Source/Core/PolylineVolumeGeometry.js

```diff
@@ -163,7 +163,13 @@ define([
         }
 
         if (vertexFormat.tangent || vertexFormat.binormal) {
-            geometry = GeometryPipeline.computeBinormalAndTangent(geometry);
+            try {
+                geometry = GeometryPipeline.computeBinormalAndTangent(geometry);
+            } catch (e) {
+                console.log('Unable to compute tangents and binormals for polyline volume geometry');
```

Comment:
It prevents the geometry from failing if you request the tangents and binormals, both in and out of the tests.